packages/core/src/config/parse.ts

```diff
@@ -66,12 +66,25 @@ import {
 import { ChugSplashRuntimeEnvironment } from '../types'
 
 class InputError extends Error {
-  constructor(message) {
+  constructor(message: string) {
     super(message)
     this.name = 'InputError'
   }
 }
 
+let validationErrors = false
+
+const logValidationError = (
+  logLevel: 'warning' | 'error' = 'warning',
```

Comment:
Is this the same as `logLevel: 'warning' | 'error'`? It appears to be when I hover over it in VSCode. If so, could simplify it